spk/lua/src/service-setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+# Package
+PACKAGE="lua"
+DNAME="Lua"
+
+
+
+service_postinst ()
+{
+	echo "Creating startup scripts for Lua executables" >> ${INST_LOG}
+	for executable in ${SYNOPKG_PKGDEST}/bin/lua*; do
+		ln -s "$executable" /usr/local/bin/$(basename "$executable")
+	done
+
+	exit 0
```

Comment:
Do not invoke `exit` in such "hook" shell function... it stops script and prevent remaining code in framework `installer` to run.